### ISSUE TITLE
Use MODIS for FGBNMS board. Or, can we have two versions?

### ISSUE THREAD

**Issue by dotis** [USF-IMARS/grafana-dashboards#15]
There is a need for MODIS data in FGB. They would like to go back to 2009 for their condition report. Is it possible to have a VIIRS version and a MODIS version?

**Comment by 7yl4r** [comment]
Yes, we can have two dashboards for this, or we could use try using a dropdown at the top to select the data source. That could be tricky if there are different technical details for the two datasets in ERDDAP though.
